Documentation/gettingstarted/encryption.rst

```diff
@@ -157,25 +183,50 @@ exec -ti <cilium pod> -- bash`` and execute the following commands:
 
 1. Install tcpdump
 
-.. code:: bash
+.. code-block:: shell-session
 
-    apt-get update
-    apt-get -y install tcpdump
+    $ apt-get update
+    $ apt-get -y install tcpdump
 
 2. Check that traffic is encrypted:
 
-.. code:: bash
+.. tabs::
+
+    .. group-tab:: GKE
+
+         .. code-block:: shell-session
+
+            $ tcpdump -n -i cilium_vxlan
+            tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
```

Comment:
I found it a bit confusing that these tabs are named "GKE", but earlier in this document there are two sets of GKE instructions, one with native mode and one with tunneling. Maybe these GKE example tabs should be just `Tunneling mode` instead?

Reply:
should we drop the tunnel mode or at least discourage it somehow. Its not a very optimal setup for us and AFAIK we don't encourage tunnel+GKE at the moment.

Reply:
I think the tunnel piece was just coming from the previous version of the doc. The main question here is how we ensure that regardless of the /other/ configuration that the user wants to use, it's clear which encryption options they should add on top.

Longer-term, the new CLI will solve this problem.